lib/game_team.rb

```diff
@@ -0,0 +1,12 @@
+class GameTeam
+   def initialize(game_team_details)
```

Comment:
I'm foreseeing that we will need an attr_reader for the attributes of a GameTeam